releases/release-5.0.1.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TiDB 5.0.1 Release Notes
+---
+
+# TiDB 5.0.1 Release Notes
+
+Release date: April 23, 2021
+
+TiDB version: 5.0.1
+
+## Compatibility Changes
+
+## New Features
+
+## Improvements
+
++ TiKV
+
+    - Use `zstd` to compress the snapshot [#10005](https://github.com/tikv/tikv/pull/10005)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - Fix the issue that the summary log is not clear [#1009](https://github.com/pingcap/br/pull/1009)
+
```

Comment:
+ PD
     - Modify score calculator to satisfy isomerous stores [#3605](https://github.com/pingcap/pd/pull/3605)
     - Avoid unexpected statistic modify after adding scatter region scheduler [#3602](https://github.com/pingcap/pd/pull/3602)